### PR TITLE
Command line config tools

### PR DESCRIPTION
**Pull request by eslickj**
Even though nobody asked for it, I put together a little utility for idaes configuration from the command line.   I also sped up the imports for the idaes command, so it doesn't run annoyingly slow.  @jsiirola I tried to move to pyomo ConfigBlocks as much as possible, but there is still a bit of an issue with logger configuration since loggers have periods in their names.  That means I have to use a dict there instead of a ConfigBlock, since ConfigBlock keys can't have periods (and I use logging.dictConfig() directly, in which keys may have periods).  @dangunter, I messed with your imports in get-examples to speed things up.  You'll probably want to take a look at that.  The main issue I had was figuring out how to mange setuptools.  I ended up just moving it in the command function call because I couldn't find another way that worked. 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by dangunter — DISMISSED** [review]
Changes to get-examples look good. Thanks for that.
I am agnostic on the other changes, but if they pass tests it's fine with me.

File: idaes/commands/base.py [hunk 1]
```diff
@@ -103,6 +103,10 @@ def copyright():
 """
     )
 
+@command_base.command(help="Show how long it takes to imoprt command modules")
```
Comment (jsiirola) [inline idaes/commands/base.py:106]: typo: "import"

**Comment by eslickj** [comment]
@dangunter and @jsiirola, I fixed the misspelled import.  If you guys finished your reviews the first time around, it's probably safe to re-approve, if not take your time.

**Review by andrewlee94 — APPROVED** [review]
I will assume this is good after @dangunter and @jsiirola's comments.

**Review by MAZamarripa — APPROVED** [review]
lgtm + (I will assume this is good after @andrewlee94, @dangunter and @jsiirola's comments)

**Pull request merged**